cypress-browser-using-plugin/README.md

```diff
@@ -0,0 +1,88 @@
+# Overview
+This is a short demo project that uses the Sealights Browser Agent and our Cypress plugin.
+Currently the Cypress plugin in this demo is based off the following repo/branch:
+[SL.Cypress.Plugin](https://github.com/kristijanstefanoski/SL.Cypress.Plugin/tree/service-integration).
+
+# Integration with Sealights - how to
+
+## 1. Configuration and build scanning
+**NOTE**: Assume that you should run all commands from this directory (/cypress-browser-using-plugin)! <br><br>
+In order to run this demo you need a valid Agent Token stored in a file ex. `sltoken.txt`.
+Then you can proceed to run `slnodejs` config command to generate a new `build session id`:
+```bash
+npx slnodejs config --tokenfile sltoken.txt --appName "Browser Example" --branch "master" --build 1.0.0
+```
+If this command ran successfully you should have `buildSessionId` file in the same folder and can continue to scan the build:
+```bash
+npx slnodejs scan --workspacepath ./calculator-app --tokenfile sltoken.txt --buildsessionidfile buildSessionId --scm none --instrumentForBrowsers --enableOpenTelemetry --outputpath "sl_web"
+```
+**IMPORTANT**: Make sure you are running `slnodejs >= 6.1.327` with `npx`, to clear cache use `npx clear-npx-cache`.
+
+After a successful build can you should have a resulting `sl_web` folder under this one and can continue with the steps bellow.
+
+**IMPORTANT:** Do not delete `sltoken.txt` of `buildSessionId` files after the build scan, they are used in the following steps.
+
```

Comment:
probably better to refer to Confluence here because theoretically this flow can be changed and noone will update it here))